README.md

```diff
@@ -37,22 +39,12 @@ projects:
 
 See [XCHammerConfig.swift](https://github.com/pinterest/xchammer/blob/master/Sources/XCHammer/XCHammerConfig.swift) for detailed documentation.
```

Comment:
I'm still on the fence if the above example is too redundant with having a sample.. It should probably include `**` as paths too BTW.